src/cloudinary-image.component.ts

```diff
@@ -145,4 +152,16 @@ export class CloudinaryImage
     })
     this.placeholderComponent.options = placeholderOptions;
   }
+
+  accessibilityModeHandler() {
+    const accessibilityEffect = {
+      'darkmode': {effect: 'tint:75:black'},
+      'brightmode': {effect: 'tint:50:white'},
+      'monochrome': {effect: 'grayscale'},
+      'colorblind': {effect: 'assist_colorblind'}
+    }
+    let transformation = [].concat.apply([], [this.options, accessibilityEffect[this.accessibility]]);
```

Comment:
@strausr , is this the most elegant way of doing this? 
It's ugly!

Reply:
:/ this is the same convention we used in the placeholder component- https://github.com/cloudinary/cloudinary_angular/blob/master/src/cloudinary-placeholder.component.ts#L42

Reply:
@strausr, not a good reason.
Maybe we can do it more elegantly there as well?